src/main/kotlin/lotto/LottoTicket.kt

```diff
@@ -1,3 +1,29 @@
 package lotto
 
-class LottoTicket(val ticket: List<Int>)
+import kotlin.random.Random
+
+class LottoTicket constructor() {
+    var ticket: List<Int>
```

Comment:
기존 코드가 더 깔끔해 보입니다.
```suggestion
class LottoTicket(val ticket: List<Int>)
```

그리고 요구사항인 모든 원시값과 문자열을 포장한다. 을 지켜볼까요?

`List<Int>` -> `List<LottoNumber>` 처럼 변할 수 있겠네요.